File: server/API/Models/DTOs/UserDto.cs
namespace API.Models.DTOs
{
    public class UserDto
    {
        public int UserId { get; set; }
        public string UserName { get; set; }
        public string Password { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public int YearsExperience { get; set; }
        public string CareerSummary { get; set; }
        public string Title { get; set; }
        public string ImageUrl { get; set; }
    }

    public class UserSkillDto
    {
        public SkillDto Skill { get; set; }

        public int Level { get; set; }
        public int ProgressPercent { get; set; }
    }
}

File: client/src/app/shared/models/dtos/user_dto.ts
export interface User extends UserInfo {
  userId: string;
}

export interface UserInfo {
  userName: string;
  firstName: string;
  lastName: string;
  imageUrl: string;
  yearsExperience: number;
  title: string;
  technicalSummary: string;
  currentEmployer: string;
  yearsAtEmployer: number;
  jobDescription: string;
}
